tfjs-backend-cpu/src/shared.ts

```diff
@@ -19,6 +19,7 @@
 export {simpleAbsImpl} from './kernels/Abs';
 export {addImpl} from './kernels/Add';
 export {bincountImpl, bincountReduceImpl} from './kernels/Bincount_impl';
+export {bitwiseAndImpl} from './kernels/BitwiseAnd';
```

Comment:
Nice catch. Adding it here means other backend could use this kernel implementation. We could discuss this later when implementing this kernel for other backends.